arches/app/models/resource.py

```diff
@@ -215,9 +232,13 @@ def get_root_ontology(self):
 
         """
         root_ontology_class = None
-        graph_nodes = models.Node.objects.filter(graph_id=self.graph_id).filter(istopnode=True)
-        if len(graph_nodes) > 0:
-            root_ontology_class = graph_nodes[0].ontologyclass
+        try:
+            graph_node = SimpleNamespace(**next((x for x in self.serialized_graph["nodes"] if x["istopnode"] == True), None))
```

Comment:
Hey @aarongundel, could you change `==` to `is` appease Stickler?

Reply:
old habits!